cloudinit/reporting/handlers.py

```diff
@@ -113,6 +113,8 @@ class HyperVKvpReportingHandler(ReportingHandler):
     https://technet.microsoft.com/en-us/library/dn798287.aspx#Linux%20guests
     """
     HV_KVP_EXCHANGE_MAX_VALUE_SIZE = 2048
+    # The maximum value size expected in Azure
```

Comment:
Is there any public facing documentation URL we can reference here so we know if this changes in the future? It seems the above link does not provide sizing guidelines for kvp messages.

Reply:
Unfortunately, We don't currently have any documentation covering this limit.